mscore/scoreaccessibility.cpp

```diff
@@ -294,6 +294,12 @@ void ScoreAccessibility::makeReadable(QString& s)
             { "♯", tr(" sharp") },
             { "𝄫", tr(" double flat") },
             { "𝄪", tr(" double sharp") },
+            { "", tr("sixteenth note") },
+            { "", tr("eighth note") },
+            { "", tr("quarter note") },
+            { "", tr("half note") },
+            { "", tr("whole note") },
+            { "", tr("dot") },
```

Comment:
Maybe be better use `\uxxxx` notation here?

Reply:
Probably so, especially given I really should support both the Unicode and SMuFL versions.  And rests.

Right now, though, I'm trying to understand what is going on with these symbols, it seems copy/paste automatically converts one form to the other, but one version renders as Chinese on musescore.com.  I will investigate a bit more and file an issue when I can characterize more fully.